db/DB.js

```diff
@@ -8,22 +10,29 @@ const knexConfig = require("./knexfile");
 const HIBP = require("../hibp");
 const getSha1 = require("../sha1-utils");
 
-const knex = Knex(knexConfig);
+const knex = Knex(knexConfig[AppConstants.NODE_ENV]);
 
 
 const DB = {
+  async getSubscriberByEmailAndToken(email, token) {
+    const res = await knex("subscribers")
+      .where("verification_token", "=", token)
+      .andWhere("email", "=", email);
+
+    return res[0];
```

Comment:
Sorry, was `res[0]` a thing? Or did you have a TODO there before to use `.first()` API?

https://knexjs.org/#Builder-first

Reply:
`first` only works on `select` queries/functions. Doesn't work on these `update` and `insert`